static/css/cf-enhancements.less

```diff
@@ -146,7 +148,7 @@
     - cfgov-cf-enhancements
 */
 
-.respond-to-retina(@rules) {
+.respond-to-dpi(@rules) {
     @media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
```

Comment:
I see what you're saying. I thought the issue was the retina was too Apple specific.